website/static/js/myProjects.js

```diff
@@ -1635,7 +1635,9 @@ var Breadcrumbs = {
                                     });
                                 },
                                 trackingCategory: 'myProjects',
-                                trackingAction: 'add-component'
+                                trackingAction: 'add-component',
+                                contributors: Array.from(linkObject.data.contributorSet),
```

Comment:
Can you explain why `Array.from` is necessary here? What data type is `contributorSet`?